.github/workflows/auto-assign-issue.yml

```
@@ -11,9 +11,20 @@ jobs:
       issues: write
 
     steps:
+      - name: 'Get repository collaborators'
+        id: get-collaborators
+        run: |
+          COLLABORATORS=$(gh api repos/${{ github.repository }}/collaborators --jq 'map(.login) | map(select(. != "hellices")) | join(", ")')
+          echo "COLLABORATORS=${COLLABORATORS}" >> $GITHUB_ENV
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: 'Check COLLABORATORS Value'
+        run: echo "COLLABORATORS=${{ env.COLLABORATORS }}"
+
       - name: 'Auto-assign Issue'
-        uses: pozil/auto-assign-issue@v2.2.0
+        uses: pozil/auto-assign-issue@v2
```

```suggestion
        uses: pozil/auto-assign-issue@v2.2.0
```

이건 최신으로 테스트 안해보긴했는데.. 음...